src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/config/MessagingMultiNodeTestBase.kt

```diff
@@ -268,6 +268,7 @@ abstract class MessagingMultiNodeTestBase : IntegrationTestBase() {
     purgeQueueAndDlq(prisonMergeEventsQueue)
     purgeQueueAndDlq(reclusterEventsQueue)
     expectNoMessagesOnQueueOrDlq(reclusterEventsQueue)
+    stubPersonMatch()
```

Comment:
The code should not be this unpredictable. We should know when we are going to call match, and it surely cannot be on every message we process. e.g. the ones which cause 500 errors
Doing this leads to unsatisified mock expectations (which we don't check a lot if at all) but I feel we need to look to be deterministic in our testing wherever we can